docs/book/component-guide/alerters/discord.md

```diff
@@ -97,18 +114,64 @@ def my_formatter_step(artifact_to_be_communicated) -> str:
     return f"Here is my artifact {artifact_to_be_communicated}!"
 
 
+@step
+def conditional_step(artifact, approved: bool) -> None:
```

Comment:
not sure this is a `conditional` step.. maybe a step that has a condition in it? :-)

Reply:
Ok I renamed these. Probably was confusing indeed!